prov/rxm/src/rxm_cq.c

```diff
@@ -316,6 +316,7 @@ static int rxm_rndv_tx_finish(struct rxm_ep *rxm_ep, struct rxm_tx_rndv_buf *tx_
 
 	RXM_LOG_STATE_TX(FI_LOG_CQ, tx_buf, RXM_RNDV_FINISH);
 	tx_buf->hdr.state = RXM_RNDV_FINISH;
+	tx_buf->conn->rndv_tx_credits++;
```

Comment:
Are we holding the ep lock here?

Reply:
We have ep lock in the rxm_ep_progress function. This function is part of the call stack that begins with rxm_ep_progress. I think we would probably not need cmap->lock here as well because this wouldn't be called before the first rendezvous request is sent.